### PR TITLE
✨ Source Facebook Marketing, Salesforce, Shopify, Stripe: add maxSecondsBetweenMessages in metadata

### PR DESCRIPTION
**Pull request by strosek**
Certified connectors are required to have `maxSecondsBetweenMessages` set in metadata file.
`maxSecondsBetweenMessages` is the longest time frame (in seconds) of API requests limits reset for endpoints used by connector. [Issue](https://app.zenhub.com/workspaces/critical-connectors-659c693d906fcd00178900cf/issues/gh/airbytehq/airbyte-internal-issues/6779)

## What
`maxSecondsBetweenMessages` value for the following connectors was added:

source-stripe [reference](https://docs.stripe.com/rate-limits)
source-facebook-marketing [reference](https://developers.facebook.com/docs/marketing-apis/rate-limiting)
source-salesforce [reference](https://developer.salesforce.com/docs/atlas.en-us.salesforce_app_limits_cheatsheet.meta/salesforce_app_limits_cheatsheet/salesforce_app_limits_platform_api.htm)
source-shopify [reference](https://shopify.dev/docs/api/usage/rate-limits)

## How
Files `metadata.yaml` for connectors mentioned above were updated by adding `maxSecondsBetweenMessages` value.

## 🚨 User Impact 🚨
No breaking changes

### CODE REVIEW

**Review by maxi297 — APPROVED** [review]
LGTM! Conditional approval on @davinchia 's approval.

A couple of things regarding the documentation of PR:
* Can you link the ticket here? We have multiple repos with issues so having the link will help relate to the actual issues 
* Can we update the ticket based on [those comments](https://airbytehq-team.slack.com/archives/C06H4LJRK1Q/p1711379371421849?thread_ts=1711377477.240559&cid=C06H4LJRK1Q) to explain why we're not doing the others in the ticket (yet)

**Comment by maxi297** [comment]
I'm fine with Davin not approving following our conversations. Feel free to merge any time you are comfortable with!

File: airbyte-integrations/connectors/source-shopify/metadata.yaml [hunk 1]
```diff
@@ -17,6 +17,7 @@ data:
   githubIssueLabel: source-shopify
   icon: shopify.svg
   license: ELv2
+  maxSecondsBetweenMessages: 60
```
Comment (maxi297) [inline airbyte-integrations/connectors/source-shopify/metadata.yaml:20]: Checking shopify, which rate limit is by minute? I only see some per second

Comment (strosek) [reply airbyte-integrations/connectors/source-shopify/metadata.yaml:20]: You're right. I didn't double check the ones not included originally. I'll change it to 1s, and FB to 300s.

**Pull request closed without merging**